reporting/test/webrequest-pipeline/index.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WhoTracks.Me
+ * https://ghostery.com/whotracksme
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { expect } from 'chai';
+
+import WebRequestPipeline from '../../src/webrequest-pipeline/index.js';
+
+describe('WebRequestPipeline', function () {
+  beforeEach(function () {
+    chrome.flush();
+    chrome.storage.session.get.yields({});
+  });
+
+  after(function () {
+    chrome.flush();
+  });
+
+  context('PageStore', function () {
+    it('starts empty', async function () {
+      const pipeline = new WebRequestPipeline();
+      await pipeline.init();
+      expect(pipeline.pageStore.tabs).to.have.property('size', 0);
+    });
+
+    context('on webRequest.onBeforeRequest', function () {
+      it('creates a page', async function () {
+        const pipeline = new WebRequestPipeline();
+        pipeline.init();
```

Comment:
This look inconsistent with the previous use of
```
await pipeline.init()
```

Reply:
currently WebrequestPipeline init is sync - I didn't want to introduce a change in this PR.

Reply:
It is just a bit confusing that the test before initializes like that:
```
    const pipeline = new WebRequestPipeline();
    await pipeline.init();
```  

and the next test uses:
```
    const pipeline = new WebRequestPipeline();
    pipeline.init();
```  

Since WebRequestPipeline.init is indeed synchronous (https://github.com/whotracksme/webextension-packages/blob/398a5b5540c0ec42ec01617d2fbee261c101ae5e/reporting/src/webrequest-pipeline/index.js#L103), I would suggest to remove the unnecessary `await` in the previous test.

Also, we could make the tests synchronous. It is not technically wrong to mark them async or await the synchronous init, but I find it non-intuitive when reading the tests (since they are testing a synchronous API).

Reply:
ah, you're right.